frontend/src/app/Settings/Users/UserDetail.tsx

```diff
@@ -148,43 +358,13 @@ const UserDetail: React.FC<Props> = ({
       <div>
         <h2 className="display-inline-block margin-top-1 margin-bottom-0 user-name-header">
           {displayFullName(user.firstName, user.middleName, user.lastName)}
-          {user?.id === loggedInUser.id ? (
-            <span className="usa-tag margin-left-1 bg-base-lighter text-ink">
-              YOU
-            </span>
-          ) : null}
+          {displayYou()}
         </h2>
-        <div className="user-status-subheader">{statusText}</div>
+        <div className="user-status-subheader">{getUserStatusText()}</div>
```

Comment:
I would either make this a subcomponent `<UserStatusSubheading/>` or make this return the entire JSX node `{getUserStatusSubheading()}` similar to line 361 with the `displayYou()`. The name getUserStatusText is quiet not right because what is being return is actually DOM not just text.